internal/stack/serverless.go

```diff
@@ -261,7 +270,8 @@ func (sp *serverlessProvider) BootUp(options Options) error {
 		}
 
 		logger.Infof("Creating agent policy")
-		err = project.CreateAgentPolicy(options.StackVersion, sp.kibanaClient)
+		err = project.CreateAgentPolicy(options.StackVersion, sp.kibanaClient, settings.LogstashEnabled)
```

Comment:
Decide here about the output, so `CreateAgentPolicy` doesn't need to know about logstash.

```suggestion
		outputID := ""
		if settings.LogstashEnabled {
			outputID = FleetLogstashOutput
		}
		err = project.CreateAgentPolicy(options.StackVersion, sp.kibanaClient, outputID)
```

Reply:
Ok.. There still needs to be a check for `outputId != ""` in `createAgentPolicy` though

Reply:
Well, we can set the empty value and it should not affect the resulting JSON.